third_party/terraform/tests/resource_dataproc_job_test.go

```diff
@@ -22,21 +22,22 @@ type jobTestField struct {
 	gcp_attr interface{}
 }
 
-func TestAccDataprocJob_failForMissingJobConfig(t *testing.T) {
-	t.Parallel()
-
-	resource.Test(t, resource.TestCase{
-		PreCheck:     func() { testAccPreCheck(t) },
-		Providers:    testAccProviders,
-		CheckDestroy: testAccCheckDataprocJobDestroy,
-		Steps: []resource.TestStep{
-			{
-				Config:      testAccDataprocJob_missingJobConf(),
-				ExpectError: regexp.MustCompile("You must define and configure exactly one xxx_config block"),
-			},
-		},
-	})
-}
+// TODO (mbang): Test `ExactlyOneOf` here
```

Comment:
With using `ExactlyOneOf`, each field will produce an error and it seemed, last I tested this, that the order of those errors wasn't consistent, so I need to look back into how to check this.